Please dont kill the script while its working
Please dont kill the script while its working
Please dont kill the script while its working
Please dont kill the script while its working
Please dont kill the script while its working
Please dont kill the script while its working
Please dont kill the script while its working
Please dont kill the script while its working
Please dont kill the script while its working
Please dont kill the script while its working
Please dont kill the script while its working
Please dont kill the script while its working
Please dont kill the script while its working
Please dont kill the script while its working
Please dont kill the script while its working
Please dont kill the script while its working
Please dont kill the script while its working
Please dont kill the script while its working
Please dont kill the script while its working
Please dont kill the script while its working
Please dont kill the script while its working
Please dont kill the script while its working
Please dont kill the script while its working
Please dont kill the script while its working
Please dont kill the script while its working
Please dont kill the script while its working
Traceback (most recent call last):

  File "html_pinger.py", line 200, in <module>
    main()

  File "html_pinger.py", line 195, in main
    with open(file_if_in_statistics_mode, "w") as f:

PermissionError: [Errno 13] Permission denied: "/root//2017-05-13T16:20:18+00:00-Momo's Pinger-output.json"

Please dont kill the script while its working
Traceback (most recent call last):

  File "html_pinger.py", line 200, in <module>
    main()

  File "html_pinger.py", line 195, in main
    with open(file_if_in_statistics_mode, "w") as f:

PermissionError: [Errno 13] Permission denied: "/root//2017-05-13T16:21:56+00:00-Momo's Pinger-output.json"

**************************
**************************
**************************
**************************
13/05/2017
Traceback (most recent call last):

  File "html_pinger.py", line 200, in <module>
    main()

  File "html_pinger.py", line 195, in main
    with open(file_if_in_statistics_mode, "w") as f:

PermissionError: [Errno 13] Permission denied: "/root//2017-05-13T16:25:36+00:00-Momo's Pinger-output.json"

**************************
13/05/2017
Traceback (most recent call last):

  File "html_pinger.py", line 200, in <module>
    main()

  File "html_pinger.py", line 195, in main
    with open(file_if_in_statistics_mode, "w") as f:

PermissionError: [Errno 13] Permission denied: "/root//2017-05-13T16:26:26+00:00-Momo's Pinger-output.json"

**************************
Sat May 13 19:27:01 2017
Traceback (most recent call last):

  File "html_pinger.py", line 200, in <module>
    main()

  File "html_pinger.py", line 195, in main
    with open(file_if_in_statistics_mode, "w") as f:

PermissionError: [Errno 13] Permission denied: "/root//2017-05-13T16:27:01+00:00-Momo's Pinger-output.json"

Please dont kill the script while its working
Please dont kill the script while its working
**************************
Sat May 13 19:28:24 2017
Traceback (most recent call last):

  File "html_pinger.py", line 200, in <module>
    main()

  File "html_pinger.py", line 195, in main
    with open(file_if_in_statistics_mode, "w") as f:

PermissionError: [Errno 13] Permission denied: "/root/2017-05-13T16:28:24+00:00-Momo's Pinger-output.json"

Please dont kill the script while its working
Please dont kill the script while its working
Please dont kill the script while its working
Please dont kill the script while its working
Please dont kill the script while its working
Please dont kill the script while its working
Please dont kill the script while its working
Please dont kill the script while its working
Please dont kill the script while its working
Please dont kill the script while its working
Please dont kill the script while its working
Please dont kill the script while its working
Please dont kill the script while its working
**************************
Thu Jun  8 14:12:17 2017
Traceback (most recent call last):

  File "html_pinger.py", line 207, in <module>
    main()

  File "html_pinger.py", line 192, in main
    data.link_exception(error_handling_library.test_hooks.greenletException)

NameError: name 'error_handling_library' is not defined

**************************
Thu Jun  8 14:12:19 2017
Traceback (most recent call last):

  File "html_pinger.py", line 207, in <module>
    main()

  File "html_pinger.py", line 192, in main
    data.link_exception(error_handling_library.test_hooks.greenletException)

NameError: name 'error_handling_library' is not defined

**************************
Thu Jun  8 14:12:20 2017
Traceback (most recent call last):

  File "html_pinger.py", line 207, in <module>
    main()

  File "html_pinger.py", line 192, in main
    data.link_exception(error_handling_library.test_hooks.greenletException)

NameError: name 'error_handling_library' is not defined

Please dont kill the script while its working
There was an error in the greenlet: ("Expected: (<class 'str'>,)", "Got: <class 'int'>")
Please dont kill the script while its working
There was an error in the greenlet: ("Expected: (<class 'str'>,)", "Got: <class 'int'>")
Please dont kill the script while its working
There was an error in the greenlet: ("Expected: (<class 'str'>,)", "Got: <class 'int'>")
There was an error in the greenlet: unsupported operand type(s) for +: 'TooManyRedirects' and 'str'
There was an error in the greenlet: ("Expected: (<class 'str'>,)", "Got: <class 'int'>")
Exceeded 30 redirects.
There was an error in the greenlet: ("Expected: (<class 'str'>,)", "Got: <class 'int'>")
Please dont kill the script while its working
There was an error in the greenlet: ("Expected: (<class 'str'>,)", "Got: <class 'int'>")
Please dont kill the script while its working
There was an error in the greenlet: 'module' object is not callable
There was an error in the greenlet: 'module' object is not callable
There was an error in the greenlet: ("Expected: (<class 'str'>,)", "Got: <class 'int'>")
Please dont kill the script while its working
There was an error in the greenlet: name 'Greenlet' is not defined
There was an error in the greenlet: ("Expected: (<class 'str'>,)", "Got: <class 'int'>")
Please dont kill the script while its working
There was an error in the greenlet: ("Expected: (<class 'str'>,)", "Got: <class 'int'>")
Please dont kill the script while its working
There was an error in the greenlet: ("Expected: (<class 'str'>,)", "Got: <class 'int'>")
Exceeded 30 redirects.
There was an error in the greenlet: ("Expected: (<class 'str'>,)", "Got: <class 'int'>")
Exceeded 30 redirects.
There was an error in the greenlet: ("Expected: (<class 'str'>,)", "Got: <class 'int'>")
Please dont kill the script while its working
There was an error in the greenlet: ("Expected: (<class 'str'>,)", "Got: <class 'int'>")
Please dont kill the script while its working
There was an error in the greenlet: ("Expected: (<class 'str'>,)", "Got: <class 'int'>")
Please dont kill the script while its working
There was an error in the greenlet: ("Expected: (<class 'str'>,)", "Got: <class 'int'>")
Please dont kill the script while its working
There was an error in the greenlet: ("Expected: (<class 'str'>,)", "Got: <class 'int'>")
Please dont kill the script while its working
There was an error in the greenlet: ("Expected: (<class 'str'>,)", "Got: <class 'int'>")
Exceeded 30 redirects.
There was an error in the greenlet: ("Expected: (<class 'str'>,)", "Got: <class 'int'>")
Please dont kill the script while its working
**************************
Thu Jun  8 15:28:09 2017
Traceback (most recent call last):

  File "html_pinger.py", line 213, in <module>
    main()

  File "html_pinger.py", line 199, in main
    data.join()

  File "/usr/local/lib/python3.5/dist-packages/gevent/greenlet.py", line 496, in join
    result = self.parent.switch()

  File "/usr/local/lib/python3.5/dist-packages/gevent/hub.py", line 630, in switch
    return RawGreenlet.switch(self)

gevent.hub.LoopExit: ('This operation would block forever', <Hub at 0x7fe3ce2380e0 epoll default pending=0 ref=0 fileno=4>)

There was an error in the greenlet: ("Expected: (<class 'str'>,)", "Got: <class 'int'>")
Please dont kill the script while its working
There was an error in the greenlet: ("Expected: (<class 'str'>,)", "Got: <class 'int'>")
Please dont kill the script while its working
There was an error in the greenlet: ("Expected: (<class 'str'>,)", "Got: <class 'int'>")
Please dont kill the script while its working
There was an error in the greenlet: ("Expected: (<class 'str'>,)", "Got: <class 'int'>")
Exceeded 30 redirects.
There was an error in the greenlet: ("Expected: (<class 'str'>,)", "Got: <class 'int'>")
Exceeded 30 redirects.
There was an error in the greenlet: ("Expected: (<class 'str'>,)", "Got: <class 'int'>")
Exceeded 30 redirects.
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: ("Expected: (<class 'str'>,)", "Got: <class 'int'>")
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: ("Expected: (<class 'str'>,)", "Got: <class 'int'>")
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: ("Expected: (<class 'str'>,)", "Got: <class 'int'>")
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: 
There was an error in the greenlet: ("Expected: (<class 'str'>,)", "Got: <class 'int'>")
Please dont kill the script while its working
There was an error in the greenlet: ("Expected: (<class 'str'>,)", "Got: <class 'int'>")
Please dont kill the script while its working
There was an error in the greenlet: ("Expected: (<class 'str'>,)", "Got: <class 'int'>")
Please dont kill the script while its working
There was an error in the greenlet: ("Expected: (<class 'str'>,)", "Got: <class 'int'>")
Please dont kill the script while its working
There was an error in the greenlet: ("Expected: (<class 'str'>,)", "Got: <class 'int'>")
Exceeded 30 redirects.
There was an error in the greenlet: ("Expected: (<class 'str'>,)", "Got: <class 'int'>")
Exceeded 30 redirects.
There was an error in the greenlet: ("Expected: (<class 'str'>,)", "Got: <class 'int'>")
Exceeded 30 redirects.
There was an error in the greenlet: ("Expected: (<class 'str'>,)", "Got: <class 'int'>")
Please dont kill the script while its working
There was an error in the greenlet: ("Expected: (<class 'str'>,)", "Got: <class 'int'>")
Exceeded 30 redirects.
There was an error in the greenlet: ("Expected: (<class 'str'>,)", "Got: <class 'int'>")
Please dont kill the script while its working
There was an error in the greenlet: ("Expected: (<class 'str'>,)", "Got: <class 'int'>")
Please dont kill the script while its working
There was an error in the greenlet: __init__() takes 2 positional arguments but 3 were given
There was an error in the greenlet: __init__() takes 2 positional arguments but 3 were given
There was an error in the greenlet: __init__() takes 2 positional arguments but 3 were given
There was an error in the greenlet: __init__() takes 2 positional arguments but 3 were given
There was an error in the greenlet: __init__() takes 2 positional arguments but 3 were given
There was an error in the greenlet: __init__() takes 2 positional arguments but 3 were given
There was an error in the greenlet: __init__() takes 2 positional arguments but 3 were given
There was an error in the greenlet: __init__() takes 2 positional arguments but 3 were given
There was an error in the greenlet: __init__() takes 2 positional arguments but 3 were given
There was an error in the greenlet: __init__() takes 2 positional arguments but 3 were given
There was an error in the greenlet: __init__() takes 2 positional arguments but 3 were given
There was an error in the greenlet: __init__() takes 2 positional arguments but 3 were given
There was an error in the greenlet: __init__() takes 2 positional arguments but 3 were given
There was an error in the greenlet: __init__() takes 2 positional arguments but 3 were given
There was an error in the greenlet: __init__() takes 2 positional arguments but 3 were given
There was an error in the greenlet: __init__() takes 2 positional arguments but 3 were given
There was an error in the greenlet: __init__() takes 2 positional arguments but 3 were given
There was an error in the greenlet: __init__() takes 2 positional arguments but 3 were given
There was an error in the greenlet: __init__() takes 2 positional arguments but 3 were given
There was an error in the greenlet: __init__() takes 2 positional arguments but 3 were given
There was an error in the greenlet: __init__() takes 2 positional arguments but 3 were given
Please dont kill the script while its working
Please dont kill the script while its working
Please dont kill the script while its working
Please dont kill the script while its working
Please dont kill the script while its working
**************************
Wed Jun 21 13:29:05 2017
Traceback (most recent call last):

  File "pinger_refactor.py", line 214, in <module>
    main()

  File "pinger_refactor.py", line 207, in main
    stater.printData()

  File "pinger_refactor.py", line 74, in printData
    .format(OPENED_SOCKETS, CURRENT_TRY_TO_BE_OPENED_SOCKETS, OPENED_HOSTS, CURRENT_HOST, CURRENT_IN_GETHOSTBYNAME_EX, CURRENT_HOST - OPENED_HOSTS))

NameError: name 'OPENED_HOSTS' is not defined

**************************
Wed Jun 21 13:29:10 2017
Traceback (most recent call last):

  File "pinger_refactor.py", line 214, in <module>
    main()

  File "pinger_refactor.py", line 207, in main
    stater.printData()

  File "pinger_refactor.py", line 74, in printData
    .format(OPENED_SOCKETS, CURRENT_TRY_TO_BE_OPENED_SOCKETS, OPENED_HOSTS, CURRENT_HOST, CURRENT_IN_GETHOSTBYNAME_EX, CURRENT_HOST - OPENED_HOSTS))

NameError: name 'OPENED_HOSTS' is not defined

**************************
Wed Jun 21 13:29:11 2017
Traceback (most recent call last):

  File "pinger_refactor.py", line 214, in <module>
    main()

  File "pinger_refactor.py", line 207, in main
    stater.printData()

  File "pinger_refactor.py", line 74, in printData
    .format(OPENED_SOCKETS, CURRENT_TRY_TO_BE_OPENED_SOCKETS, OPENED_HOSTS, CURRENT_HOST, CURRENT_IN_GETHOSTBYNAME_EX, CURRENT_HOST - OPENED_HOSTS))

NameError: name 'OPENED_HOSTS' is not defined

**************************
Wed Jun 21 13:29:11 2017
Traceback (most recent call last):

  File "pinger_refactor.py", line 214, in <module>
    main()

  File "pinger_refactor.py", line 207, in main
    stater.printData()

  File "pinger_refactor.py", line 74, in printData
    .format(OPENED_SOCKETS, CURRENT_TRY_TO_BE_OPENED_SOCKETS, OPENED_HOSTS, CURRENT_HOST, CURRENT_IN_GETHOSTBYNAME_EX, CURRENT_HOST - OPENED_HOSTS))

NameError: name 'OPENED_HOSTS' is not defined

**************************
Wed Jun 21 13:29:15 2017
Traceback (most recent call last):

  File "pinger_refactor.py", line 214, in <module>
    main()

  File "pinger_refactor.py", line 207, in main
    stater.printData()

  File "pinger_refactor.py", line 74, in printData
    .format(OPENED_SOCKETS, CURRENT_TRY_TO_BE_OPENED_SOCKETS, OPENED_HOSTS, CURRENT_HOST, CURRENT_IN_GETHOSTBYNAME_EX, CURRENT_HOST - OPENED_HOSTS))

NameError: name 'OPENED_HOSTS' is not defined

Please dont kill the script while its working
Please dont kill the script while its working
Please dont kill the script while its working
Please dont kill the script while its working
There was an error in the greenlet: local variable 'OPENED_HOSTS' referenced before assignment
There was an error in the greenlet: local variable 'OPENED_HOSTS' referenced before assignment
There was an error in the greenlet: local variable 'OPENED_HOSTS' referenced before assignment
There was an error in the greenlet: local variable 'OPENED_HOSTS' referenced before assignment
There was an error in the greenlet: local variable 'OPENED_HOSTS' referenced before assignment
There was an error in the greenlet: local variable 'OPENED_HOSTS' referenced before assignment
There was an error in the greenlet: local variable 'OPENED_HOSTS' referenced before assignment
**************************
Wed Jun 21 13:40:05 2017
Traceback (most recent call last):

  File "pinger_refactor.py", line 210, in <module>
    main()

  File "pinger_refactor.py", line 206, in main
    data.join()

  File "/usr/local/lib/python3.5/dist-packages/gevent/greenlet.py", line 496, in join
    result = self.parent.switch()

  File "/usr/local/lib/python3.5/dist-packages/gevent/hub.py", line 630, in switch
    return RawGreenlet.switch(self)

gevent.hub.LoopExit: ('This operation would block forever', <Hub at 0x7f3792eb03d8 epoll default pending=0 ref=0 fileno=4>)

**************************
Wed Jun 21 13:42:54 2017
Traceback (most recent call last):

  File "pinger_refactor.py", line 210, in <module>
    main()

  File "pinger_refactor.py", line 206, in main
    data.join()

  File "/usr/local/lib/python3.5/dist-packages/gevent/greenlet.py", line 496, in join
    result = self.parent.switch()

  File "/usr/local/lib/python3.5/dist-packages/gevent/hub.py", line 630, in switch
    return RawGreenlet.switch(self)

gevent.hub.LoopExit: ('This operation would block forever', <Hub at 0x7f3668e613d8 epoll default pending=0 ref=0 fileno=4>)

There was an error in the greenlet: name 'join' is not defined
There was an error in the greenlet: name 'join' is not defined
There was an error in the greenlet: name 'join' is not defined
There was an error in the greenlet: name 'join' is not defined
There was an error in the greenlet: name 'join' is not defined
There was an error in the greenlet: name 'join' is not defined
There was an error in the greenlet: module 'gevent' has no attribute 'join'
There was an error in the greenlet: module 'gevent' has no attribute 'join'
There was an error in the greenlet: module 'gevent' has no attribute 'join'
There was an error in the greenlet: module 'gevent' has no attribute 'join'
There was an error in the greenlet: module 'gevent' has no attribute 'join'
There was an error in the greenlet: module 'gevent' has no attribute 'join'
There was an error in the greenlet: object of type 'Greenlet' has no len()
There was an error in the greenlet: object of type 'Greenlet' has no len()
There was an error in the greenlet: object of type 'Greenlet' has no len()
There was an error in the greenlet: object of type 'Greenlet' has no len()
There was an error in the greenlet: object of type 'Greenlet' has no len()
There was an error in the greenlet: object of type 'Greenlet' has no len()
There was an error in the greenlet: ("Expected: (<class 'str'>,)", "Got: <class 'int'>")
Please dont kill the script while its working
There was an error in the greenlet: ("Expected: (<class 'str'>,)", "Got: <class 'int'>")
Please dont kill the script while its working
Please dont kill the script while its working
Exceeded 30 redirects.
Please dont kill the script while its working
Please dont kill the script while its working
Please dont kill the script while its working
Exceeded 30 redirects.
Exceeded 30 redirects.
Exceeded 30 redirects.
Exceeded 30 redirects.
Please dont kill the script while its working
Please dont kill the script while its working
There was an error in the greenlet: string indices must be integers
Please dont kill the script while its working
There was an error in the greenlet: string indices must be integers
There was an error in the greenlet: string indices must be integers
Exceeded 30 redirects.
There was an error in the greenlet: 'items'
There was an error in the greenlet: 'items'
**************************
Fri Jun 23 15:17:51 2017
Traceback (most recent call last):

  File "html_pinger.py", line 213, in <module>
    main()

  File "html_pinger.py", line 212, in main
    print(outputter.getResult(outputFormat=args.output))

  File "/home/momo/Desktop/telebit-pro/pinger/pinger/libs/outputDataIntoDict.py", line 65, in getResult
    return eval("self.getResultIn"+(outputFormat.upper())+"()")

  File "<string>", line 1, in <module>

  File "/home/momo/Desktop/telebit-pro/pinger/pinger/libs/outputDataIntoDict.py", line 59, in getResultInJSON
    return json.dumps(self.outputDictionary, ensure_ascii=False, indent = 4)

  File "/usr/lib/python3.5/json/__init__.py", line 237, in dumps
    **kw).encode(obj)

  File "/usr/lib/python3.5/json/encoder.py", line 200, in encode
    chunks = list(chunks)

  File "/usr/lib/python3.5/json/encoder.py", line 429, in _iterencode
    yield from _iterencode_dict(o, _current_indent_level)

  File "/usr/lib/python3.5/json/encoder.py", line 403, in _iterencode_dict
    yield from chunks

  File "/usr/lib/python3.5/json/encoder.py", line 403, in _iterencode_dict
    yield from chunks

  File "/usr/lib/python3.5/json/encoder.py", line 403, in _iterencode_dict
    yield from chunks

  File "/usr/lib/python3.5/json/encoder.py", line 403, in _iterencode_dict
    yield from chunks

  File "/usr/lib/python3.5/json/encoder.py", line 403, in _iterencode_dict
    yield from chunks

  File "/usr/lib/python3.5/json/encoder.py", line 339, in _iterencode_dict
    raise ValueError("Circular reference detected")

ValueError: Circular reference detected

**************************
Fri Jun 23 15:19:13 2017
Traceback (most recent call last):

  File "html_pinger.py", line 213, in <module>
    main()

  File "html_pinger.py", line 212, in main
    print(outputter.getResult(outputFormat=args.output))

  File "/home/momo/Desktop/telebit-pro/pinger/pinger/libs/outputDataIntoDict.py", line 65, in getResult
    return eval('self.getResultIn'+(outputFormat.upper())+'()')

  File "<string>", line 1, in <module>

  File "/home/momo/Desktop/telebit-pro/pinger/pinger/libs/outputDataIntoDict.py", line 59, in getResultInJSON
    return json.dumps(self.outputDictionary, ensure_ascii=False, indent = 4)

  File "/usr/lib/python3.5/json/__init__.py", line 237, in dumps
    **kw).encode(obj)

  File "/usr/lib/python3.5/json/encoder.py", line 200, in encode
    chunks = list(chunks)

  File "/usr/lib/python3.5/json/encoder.py", line 429, in _iterencode
    yield from _iterencode_dict(o, _current_indent_level)

  File "/usr/lib/python3.5/json/encoder.py", line 403, in _iterencode_dict
    yield from chunks

  File "/usr/lib/python3.5/json/encoder.py", line 403, in _iterencode_dict
    yield from chunks

  File "/usr/lib/python3.5/json/encoder.py", line 403, in _iterencode_dict
    yield from chunks

  File "/usr/lib/python3.5/json/encoder.py", line 403, in _iterencode_dict
    yield from chunks

  File "/usr/lib/python3.5/json/encoder.py", line 403, in _iterencode_dict
    yield from chunks

  File "/usr/lib/python3.5/json/encoder.py", line 339, in _iterencode_dict
    raise ValueError("Circular reference detected")

ValueError: Circular reference detected

**************************
Fri Jun 23 15:20:40 2017
Traceback (most recent call last):

  File "html_pinger.py", line 213, in <module>
    main()

  File "html_pinger.py", line 212, in main
    print(outputter.getResult(outputFormat=args.output))

  File "/home/momo/Desktop/telebit-pro/pinger/pinger/libs/outputDataIntoDict.py", line 66, in getResult
    return eval('self.getResultIn'+ outputFormat.upper() +'()')

  File "<string>", line 1, in <module>

  File "/home/momo/Desktop/telebit-pro/pinger/pinger/libs/outputDataIntoDict.py", line 59, in getResultInJSON
    return json.dumps(self.outputDictionary, ensure_ascii=False, indent = 4)

  File "/usr/lib/python3.5/json/__init__.py", line 237, in dumps
    **kw).encode(obj)

  File "/usr/lib/python3.5/json/encoder.py", line 200, in encode
    chunks = list(chunks)

  File "/usr/lib/python3.5/json/encoder.py", line 429, in _iterencode
    yield from _iterencode_dict(o, _current_indent_level)

  File "/usr/lib/python3.5/json/encoder.py", line 403, in _iterencode_dict
    yield from chunks

  File "/usr/lib/python3.5/json/encoder.py", line 403, in _iterencode_dict
    yield from chunks

  File "/usr/lib/python3.5/json/encoder.py", line 403, in _iterencode_dict
    yield from chunks

  File "/usr/lib/python3.5/json/encoder.py", line 403, in _iterencode_dict
    yield from chunks

  File "/usr/lib/python3.5/json/encoder.py", line 403, in _iterencode_dict
    yield from chunks

  File "/usr/lib/python3.5/json/encoder.py", line 339, in _iterencode_dict
    raise ValueError("Circular reference detected")

ValueError: Circular reference detected

**************************
Fri Jun 23 15:20:54 2017
Traceback (most recent call last):

  File "html_pinger.py", line 213, in <module>
    main()

  File "html_pinger.py", line 212, in main
    print(outputter.getResult(outputFormat=args.output))

  File "/home/momo/Desktop/telebit-pro/pinger/pinger/libs/outputDataIntoDict.py", line 66, in getResult
    return eval('self.getResultIn'+ outputFormat.upper() +'()')

  File "<string>", line 1, in <module>

  File "/home/momo/Desktop/telebit-pro/pinger/pinger/libs/outputDataIntoDict.py", line 60, in getResultInJSON
    return json.dumps(self.outputDictionary, ensure_ascii=False, indent = 4)

  File "/usr/lib/python3.5/json/__init__.py", line 237, in dumps
    **kw).encode(obj)

  File "/usr/lib/python3.5/json/encoder.py", line 200, in encode
    chunks = list(chunks)

  File "/usr/lib/python3.5/json/encoder.py", line 429, in _iterencode
    yield from _iterencode_dict(o, _current_indent_level)

  File "/usr/lib/python3.5/json/encoder.py", line 403, in _iterencode_dict
    yield from chunks

  File "/usr/lib/python3.5/json/encoder.py", line 403, in _iterencode_dict
    yield from chunks

  File "/usr/lib/python3.5/json/encoder.py", line 403, in _iterencode_dict
    yield from chunks

  File "/usr/lib/python3.5/json/encoder.py", line 403, in _iterencode_dict
    yield from chunks

  File "/usr/lib/python3.5/json/encoder.py", line 403, in _iterencode_dict
    yield from chunks

  File "/usr/lib/python3.5/json/encoder.py", line 339, in _iterencode_dict
    raise ValueError("Circular reference detected")

ValueError: Circular reference detected

Exceeded 30 redirects.
Exceeded 30 redirects.
There was an error in the greenlet: 'request_count'
There was an error in the greenlet: 'request_interval'
There was an error in the greenlet: 'HtmlPinger' object has no attribute 'item'
There was an error in the greenlet: 'HtmlPinger' object has no attribute 'request_interval'
There was an error in the greenlet: HTTPConnectionPool(host='msftncsi.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f4268b365c0>: Failed to establish a new connection: [Errno -5] No address associated with hostname',))
There was an error in the greenlet: HTTPConnectionPool(host='msftncsi.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7fb5fbca9550>: Failed to establish a new connection: [Errno -5] No address associated with hostname',))
There was an error in the greenlet: HTTPConnectionPool(host='msftncsi.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f3f641905c0>: Failed to establish a new connection: [Errno -5] No address associated with hostname',))
There was an error in the greenlet: name 'idx' is not defined
There was an error in the greenlet: 'idx'
There was an error in the greenlet: 'idx'
There was an error in the greenlet: 'idx'
There was an error in the greenlet: 'idx'
There was an error in the greenlet: 'idx'
There was an error in the greenlet: '1.packet_loss'
There was an error in the greenlet: '17.packet_loss'
There was an error in the greenlet: '17.packet_loss'
There was an error in the greenlet: '17.packet_loss'
**************************
Fri Jun 23 16:59:02 2017
Traceback (most recent call last):

  File "html_pinger.py", line 13, in <module>
    from libs.outputDataIntoDict import Outputter

  File "/home/momo/Desktop/telebit-pro/pinger/pinger/libs/outputDataIntoDict.py", line 58

    items[] = output

          ^

SyntaxError: invalid syntax

**************************
Fri Jun 23 16:59:03 2017
Traceback (most recent call last):

  File "html_pinger.py", line 13, in <module>
    from libs.outputDataIntoDict import Outputter

  File "/home/momo/Desktop/telebit-pro/pinger/pinger/libs/outputDataIntoDict.py", line 58

    items[] = output

          ^

SyntaxError: invalid syntax

There was an error in the greenlet: '17.packet_loss'
There was an error in the greenlet: 'list' object has no attribute 'items'
There was an error in the greenlet: name 'v' is not defined
'dict' object has no attribute 'username'
'dict' object has no attribute 'username'
**************************
Tue Jun 27 11:22:48 2017
Traceback (most recent call last):

  File "html_pinger.py", line 216, in <module>
    main()

  File "html_pinger.py", line 215, in main
    print(outputter.getResult())

TypeError: getResult() missing 1 required positional argument: 'outputFormat'

'dict' object has no attribute 'username'
'dict' object has no attribute 'username'
**************************
Tue Jun 27 11:22:55 2017
Traceback (most recent call last):

  File "html_pinger.py", line 216, in <module>
    main()

  File "html_pinger.py", line 215, in main
    print(outputter.getResult())

TypeError: getResult() missing 1 required positional argument: 'outputFormat'

'dict' object has no attribute 'username'
'dict' object has no attribute 'username'
**************************
Tue Jun 27 11:23:12 2017
Traceback (most recent call last):

  File "html_pinger.py", line 216, in <module>
    main()

  File "html_pinger.py", line 215, in main
    print(outputter.getResult())

  File "/home/momo/Desktop/telebit-pro/pinger/pinger/libs/outputDataIntoDict.py", line 64, in getResult
    return self.getResultInJson()

AttributeError: 'Outputter' object has no attribute 'getResultInJson'

'dict' object has no attribute 'username'
'dict' object has no attribute 'username'
**************************
Tue Jun 27 11:23:13 2017
Traceback (most recent call last):

  File "html_pinger.py", line 216, in <module>
    main()

  File "html_pinger.py", line 215, in main
    print(outputter.getResult())

  File "/home/momo/Desktop/telebit-pro/pinger/pinger/libs/outputDataIntoDict.py", line 64, in getResult
    return self.getResultInJson()

AttributeError: 'Outputter' object has no attribute 'getResultInJson'

'dict' object has no attribute 'username'
'dict' object has no attribute 'username'
**************************
Tue Jun 27 11:23:16 2017
Traceback (most recent call last):

  File "html_pinger.py", line 216, in <module>
    main()

  File "html_pinger.py", line 215, in main
    print(outputter.getResult())

  File "/home/momo/Desktop/telebit-pro/pinger/pinger/libs/outputDataIntoDict.py", line 64, in getResult
    return self.getResultInJson()

AttributeError: 'Outputter' object has no attribute 'getResultInJson'

'dict' object has no attribute 'username'
'dict' object has no attribute 'username'
'dict' object has no attribute 'username'
'dict' object has no attribute 'username'
There was an error in the greenlet: 'dict' object has no attribute 'timestamp'
'dict' object has no attribute 'username'
'dict' object has no attribute 'username'
There was an error in the greenlet: 'dict' object has no attribute 'timestamp'
'dict' object has no attribute 'username'
'dict' object has no attribute 'username'
There was an error in the greenlet: 'breakthrough'
'dict' object has no attribute 'username'
'dict' object has no attribute 'username'
'dict' object has no attribute 'username'
'dict' object has no attribute 'username'
'dict' object has no attribute 'username'
'dict' object has no attribute 'username'
'dict' object has no attribute 'username'
'dict' object has no attribute 'username'
'dict' object has no attribute 'username'
'dict' object has no attribute 'username'
'dict' object has no attribute 'username'
'dict' object has no attribute 'username'
